DQM/SiPixelPhase1Config/python/SiPixelPhase1OnlineDQM_cff.py

```diff
@@ -97,6 +92,7 @@
  + SiPixelPhase1TrackResidualsHarvester
  + RunQTests_online
  + SiPixelPhase1SummaryOnline
+ + SiPixelPhase1EfficiencyExtras
```

Comment:
Since this is affecting Online DQM but we cannot test it at present at P5 because it runs 11_3_X, have you tested it on your side too? When would you need these plots in Online, for CRAFT in August? Thanks

Reply:
Checking with group conveners. Our development was in 11_1_X, but we haven't explicitly tested Online. The plots are not urgent for CRAFT.

Reply:
@jfernan2 in principle this can wait till CRAFT. Jacob is testing with the online client privately now.

Reply:
OK thanks. I was asking because this PR is in 12_0_X and 12_0_0 should be ready by the end of August, since CMSSW_12_0_0 is expected to be used in the LHC beam test scheduled by the last week of September. So I am afraid that not for CRAFT...

Reply:
I guess we should just backport it, no?

Reply:
@jfernan2 Removed this Harvester from online since it is only needed in offline!